src/pages/DonateWidget/parseConfig.ts

```diff
@@ -3,10 +3,8 @@ import { DonaterConfigFromWidget, WidgetURLSearchParams } from "types/widget";
 import { number, object, string } from "yup";
 
 const schema = object<any, SchemaShape<WidgetURLSearchParams>>({
-  advancedOptionsDisplay: string()
-    .required()
-    .oneOf(["hidden", "collapsed", "expanded"]),
-  isDescriptionTextShown: string().required().oneOf(["true", "false"]),
+  splitDisabled: string().oneOf(["true", "false"]),
+  isDescriptionTextShown: string().oneOf(["true", "false"]),
```

Comment:
```suggestion
  isDescriptionTextShown: string().required().oneOf(["true", "false"]),
```

@misicnenad, @SovereignAndrey, for additional context, we previously opt for not showing widget when widget url doesn't conform to that generated by configurer.
* https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2720 

That is, we opt out of `even if incorrect, still render with default value + error message`

If we change our schema to be lax, it's easy for incorrect config to pass (default to fallback value).

**Sample invalid config (doesn't matter how it end up like this)**
```
splitDisable=true&isTextShown=true&liquidSplit=100
```

would fallback silently to a config:
```
 isDescriptionTextShown: false,
 isSplitDisabled: false,
 liquidSplitPct: 50,
```

Are we now okay with this?

Reply:
The user would see that their widget doesn't look like they had expected and would go back and recreate the code snippet.
If they don't recreate it, they either:
1. **Are ok with the widget UI** - thus there is nothing more to do here, as the user has the behavior they're fine with.
2. **Didn't verify their widget looks like they expected** - this would mean they didn't even bother to open the widget they added to the page and would've most likely missed any error message we could've provided. At some point they would realize their error and would go back and regenerate the code snippet.

Reply:
> The user would see that their widget doesn't look like they had expected and would go back and recreate the code snippet.

So we assume users, with no help from error messages, just knows what to do when something unexpected happens. What's the point of having schema and error handling then? 

> Are ok with the widget UI - thus there is nothing more to do here, as the user has the behavior they're fine with.

This doesn't make any sense. user configure his widget with a particular intent - we could not just assume that user  would settle for anything different

> Didn't verify their widget looks like they expected - this would mean they didn't even bother to open the widget 

This also doesn't make sense, you put something in your page and don't even bother to check if it's the way as expected?

Reply:
> This also doesn't make sense, you put something in your page and don't even bother to check if it's the way as expected?

That is *exactly* my point. They *would* check and see that something is not right. If you were messing around with the generated code snippet and something is not right, tthe natural thing to do next is to go back and recreate the snippet.

Reply:
Closing this thread as we're having the same discussion in the places.

Called on Andrey to make the decision on how to proceed.

Reply:
@misicnenad,
> They would check and see that something is not right. If they were messing around with the generated code snippet and something turned out wrong, the natural thing to do next is to go back and recreate the snippet.

> the natural thing to do

could you please respond to this? 
> So we assume users, with no help from error messages, just knows what to do when something unexpected happens. What's the point of having schema and error handling then?

By making our parser more forgiving, we are now trusting that our users (`normal` not `savvy`) given `UNEXPECTED` behavior, just knows what do.

So why did we opt out of fallbacks then in #2720 ?  you proposed against this right?